src/helpers/mappingHeader.ts

```diff
@@ -33,18 +34,16 @@ export function getMappingHeaderName(header: IMappingHeader): string {
     }
 }
 
-export function hasMappingHeaderIdentifier(header: IMappingHeader): boolean {
-    return isMappingHeaderAttribute(header) || isMappingHeaderMeasureItem(header);
-}
-
 export function getMappingHeaderIdentifier(header: IMappingHeader): string {
     if (isMappingHeaderAttribute(header)) {
         return header.attributeHeader.identifier;
     }
     if (isMappingHeaderMeasureItem(header)) {
         return header.measureHeaderItem.identifier;
     }
-
+    if (isMappingHeaderTotal(header)) {
+        return '';
```

Comment:
Where is this used? Wouldn't `undefined` be cleaner as returned value?

Reply:
I found the root cause and removed this code